Sources/NIOHTTP1/HTTPEncoder.swift

```diff
@@ -101,7 +101,7 @@ private enum BodyFraming {
 ///
 /// Note that for HTTP/1.0 if there is no Content-Length then the response should be followed
 /// by connection close. We require that the user send that connection close: we don't do it.
-private func sanitizeTransportHeaders(hasBody: HTTPMethod.HasBody, headers: inout HTTPHeaders, version: HTTPVersion) -> BodyFraming {
+private func correctlyFrameTransportHeaders(hasBody: HTTPMethod.HasBody, headers: inout HTTPHeaders, version: HTTPVersion) -> BodyFraming {
```

Comment:
As a linguistic note I don't think this change was needed: the word "sanitize" does not relate to the word "sanity" but to the word "sanitary". While they have the same latin root the meanings are different: sanity derives the meaning of mental health, while sanitary covers physical disease.

Regardless, this change doesn't _hurt_ in any meaningful way so I think we should just leave it.